pkg/workloadmeta/collectors/internal/remote/process_collector/process_collector.go

```diff
@@ -100,39 +100,30 @@ func (s *stream) Recv() (interface{}, error) {
 	return s.cl.Recv()
 }
 
-type streamHandler struct {
+type StreamHandler struct {
```

Comment:
since the `StreamHandler` is used after with in interface (here: https://github.com/DataDog/datadog-agent/blob/48ba568a67a85e498820b7e850224a4dab78625e/pkg/workloadmeta/collectors/internal/remote/generic.go#L59C22-L59C22). The structure can stay private, but create a function `func NewStreamHander() remote.StreamHandler`  that returns an instance of the private struct